docs/01-overview/main-areas/eventing/README.md

```diff
@@ -2,4 +2,46 @@
 title: What is Eventing in Kyma?
 ---
 
-Eventing allows you to easily integrate external applications with Kyma. Under the hood, it implements [NATS](https://docs.nats.io/) to ensure Kyma receives business events from external sources and is able to trigger business flows using Functions or services.
+With Kyma Eventing, you can focus on your business workflows and trigger them with events to implement asynchronous flows within Kyma.
+
+Kyma Eventing uses the following technology:
+- [NATS JetStream](https://docs.nats.io/) as backend within the cluster
+- [HTTP POST](https://www.w3schools.com/tags/ref_httpmethods.asp) requests to simplify sending and receiving events
+- Declarative [Subscription CR](../../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) to subscribe to events
+
+## Overview
+
+Generally, eventing consists of event producers (or publishers) and consumers (or subscribers) that send events to or receive events from an event processing backend.
+
+Kyma uses the PubSub messaging pattern, which does not send messages directly to the subscribers. Instead, it sends them to a messaging backend that filters these messages and sends them to interested subscribers. The publishers are not aware of existing and interested subscribers. The following diagram shows a high-level overview of PubSub communication:
+
+
+![PubSub](./assets/pubsub.svg)
+
+The key benefits of PubSub are:
+- Asynchronous communication between loosely coupled components
+- Easy scaling of publishers and subscribers
+
+## Eventing in Kyma in a nutshell
+
+The objective of Eventing in Kyma is to simplify the process of publishing and subscribing to events. Kyma uses proven eventing backend technology such as [NATS JetStream](https://docs.nats.io/), so that the user gets a seamless experience with their end-to-end business flow. The user does not have to implement or integrate any intermediate backend or protocol. Eventing in Kyma from a user's perspective works as follows:
+
+- Offer an HTTP end point, for example a Function to receive the events.
+- Specify the events the user is interested in using the Kyma [Subscription CR](../../../05-technical-reference/00-custom-resources/evnt-01-subscription.md).
+- Send legacy or [Cloud Events](https://cloudevents.io/) to configured HTTP end points on our [Event Publishing Proxy](https://github.com/kyma-project/kyma/tree/main/components/event-publisher-proxy) service.
```

Comment:
We want to get rid off the legacy events... lets not advertise them more then necessary ;)
```suggestion
- Send [Cloud Events](https://cloudevents.io/) or legacy events (deprecated) to configured HTTP end points on our [Event Publishing Proxy](https://github.com/kyma-project/kyma/tree/main/components/event-publisher-proxy) service.
```